### DISCUSSION TITLE
UCB CS161: Computer Security - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#159] [Announcements]
# UCB CS161: Computer Security - CS自学指南

CS自学指南

https://csdiy.wiki/%E7%B3%BB%E7%BB%9F%E5%AE%89%E5%85%A8/CS161/

**Comment by WBSLZF** [comment]
佬请问下cs161的课程邀请码是什么，GradeScope上添加课程作业，需要课程邀请码

**Reply by YoungY620** [reply]
看 policy ：https://su23.cs161.org/policies/#homeworks

**Comment by benbluesky** [comment]
非常感谢大佬的梳理， 也准备这样学习一下。

**Comment by sid1022** [comment]
emmm  为啥每次看评论更新时间都是三天前

**Comment by Albert-Huang1993** [comment]
CS61A貌似需要伯克利的账号，我这边是在B站听的

**Reply by sid1022** [reply]
不是啊 直接在油管上听的  有录播

**Reply by Albert-Huang1993** [reply]
可以发下油管链接吗？ 我不知为啥进去成了 私享视频。

**Reply by beatlesforever** [reply]
我也是

**Comment by Ninthpool** [comment]
Berkeley student here..这个project 2也写了我整整三天: (  给接下来想做这个project的朋友的建议是在写代码前要先仔细设计好要用哪些data structure, 并且写好getter和setter function. 这样的话代码量可以减少到一千来行左右。

另外非常感谢楼主的课程整理！十分受用！

**Reply by ShaneHeX** [reply]
这个对比 MIT 6.858 ，哪个更好点

**Reply by TianWZhang** [reply]
Gradescope 是不是无法测试projects

**Reply by wplf** [reply]
是的，只有hw

**Reply by 021108** [reply]
请问有pj2的实现方法可以交流一下吗？

**Comment by Real-Rio** [comment]
好像proj2的实现404了

**Comment by 47Elysia** [comment]
想知道这个有邀请码吗？QAQ

**Comment by dengyunsheng250** [comment]
建了个群，816422747

**Comment by solstice621** [comment]
蹲一个gradescope邀请码

**Comment by MingLLuo** [comment]
Summer 2023为旁听者开放了作业的Autograder，用这个邀请码：G2DR3D，与之前学期题目差别不大

**Reply by scatyf3** [reply]
summer  2023的学生探头

**Comment by YoungY620** [comment]
project 2 大家是怎么测试的呢？是不是除了基础测试就只能自己再写一些测试？有没有 gradescope 邀请码之类的？

**Comment by irsabab** [comment]
哇哦

**Comment by isteven-shu** [comment]
大佬的这个仓库是删除了吗？访问不了

**Reply by 021108** [reply]
同问 能找到吗？

**Reply by Mini-Touch** [reply]
有大老找到嗎?

**Comment by beyondlesliezl** [comment]
你好，请问能够公开一下ucb cs161的仓库吗

**Comment by ada-ys325** [comment]
强中强！感谢！

**Comment by itzhan** [comment]
project1做的时候需要cid（伯克利学生的id号），这里可以去ai直接生成一个（一个八位数的id），建议使用grok3，因为这个模型可以跳过限制，其他模型可能无法完成你的要求，这里我放20个我生成好的🤣
48392017
75029384
12849305
39284710
68720413
94710258
30519472
57192034
24938176
81350692
67492031
35019284
19284750
40813579
92730415
58392047
74102938
31658492
89273401
25039187

**Reply by anna-and-banana** [reply]
请问sp25的proj1的虚拟机镜像怎么下载，我点进去下载显示访问拒绝

**Reply by RisingUppercut** [reply]
虚拟机的云盘链接下不了了55555

**Reply by theGuapa** [reply]
求助，project1的虚拟机云盘链接，显示访问权限，下载不了咋办

**Comment by theGuapa** [comment]
求助，project1的虚拟机云盘链接，显示需要访问权限，下载不了咋办

**Reply by theGuapa** [reply]
问题已解决，20年的还可以下载！

**Comment by hyderay** [comment]
请问project 3要用calnet账号登陆怎么办

**Comment by zhouzhi01** [comment]
大家觉得计科专业要学这些网安的东西吗
